benchmarks/triton_kernels_benchmark/flash_attention_benchmark.py

```diff
@@ -571,7 +573,7 @@ def get_benchmark(
                     for z in [1, 2, 4, 8, 16, 32]
                     for (h, dhead) in [(16, 128), (32, 64)]
                     for causal in [False, True]
-                    for mode in [fa_kernel_mode]]  #
+                    for mode in [fa_kernel_mode]]
```

Comment:
needed for pre_commit
```suggestion
                    for mode in [fa_kernel_mode]]  #
```